test/3x/torch/test_logger.py

```diff
@@ -3,6 +3,21 @@
 
 from neural_compressor.common.utils import logger
 
+log_msg_lst = [
+    "call logger log function.",
```

Comment:
`3x/torch/test_logger.py`, do we need additional test for each framework?

Reply:
I have the same question.

Reply:
No, the logger tests are framework-agnostic. I will remove `3x/torch/test_logger.py` later.
More details about the impact of `3x/common` can be found here https://github.com/intel/neural-compressor/discussions/1445#discussioncomment-8227945.

Reply:
I removed it in [f290f07](https://github.com/intel/neural-compressor/pull/1583/commits/f290f07f009d0de245cdfa3726e0546ddf0b888b).